packages/documentation/src/stories/components/icon/icon.stories.tsx

```diff
@@ -1,65 +1,77 @@
-import React from 'react';
-import { Meta, Args, Story } from '@storybook/react';
+import { Args, Story } from '@storybook/react';
 import { BADGE } from '@geometricpanda/storybook-addon-badges';
 import docsPage from './icon.docs.mdx';
 
 import { forEach } from '../../../utils/react';
 
 export default {
   title: 'Components/Icon',
+  component: 'post-icon',
   parameters: {
     docs: {
       page: docsPage,
     },
     badges: [BADGE.NEEDS_REVISION],
   },
   args: {
-    'name': '1022',
-    'base': '',
-    'flip-h': false,
-    'flip-v': false,
-    'scale': 1,
-    'rotate': 0,
-    'animation': 'null',
+    name: '1022',
+    base: '',
+    flipH: false,
```

Comment:
Now that `flipH`, `flipV`, `scale`, `rotate` and `animation` have default values, that should be reflected in the `args` without manually declaring them.